specs/applicationdata/ApplicationData.md

```diff
@@ -0,0 +1,428 @@
+returns `C:\Windows\SystemTemp` for SYSTEM processes else for non-SYSTEM processes checks for the
+existence of environment variables in the following order and uses the first path found:
+
+1. The path specified by the TMP environment variable.
+2. The path specified by the TEMP environment variable.
+3. The path specified by the USERPROFILE environment variable.
+4. The Windows directory.
+
+See https://learn.microsoft.com/windows/win32/api/fileapi/nf-fileapi-gettemppath2w for more details.
+
+GetTempPath2W() is available on Windows version &gt;= 10.0.20348.0. `ApplicationData.TemporaryPath`
+performs the equivalent logic on older systems (aka Polyfill).
+
+# 4. Examples
+
+## 4.1. Packaged app using LocalPath
+
+Fabrikam packaged app creating a file in the LocalPath data store.
+
+```c#
+void SaveData(string data)
+{
+    var applicationData = ApplicationData.GetDefault();
+    var filename = Path.Combine(applicationData.LocalPath, "stuff.dat");
+    File.WriteAllText(filename, data, System.Text.UTF8Encoding);
+}
+```
+
+## 4.2. Unpackaged app using package'd app's LocalPath
+
+An unpackaged app app creating a file in the Fabrikam app's LocalPath data store.
+
+```c#
+void SaveData(string data)
+{
+    string packageFamilyName = "Fabrikam.Contoso_1234567890abc";
+    var applicationData = ApplicationData.GetForPackageFamily(packageFamilyName);
+    var filename = Path.Combine(applicationData.LocalPath, "stuff.dat");
+    File.WriteAllText(filename, data, System.Text.UTF8Encoding);
+}
+```
+
+## 4.3. Packaged app using MachinePath
+
+Fabrikam packaged app running elevated and creating a file in the LocalPath data store.
+
+```c#
+void SaveData(string data)
+{
+    var applicationData = ApplicationData.GetDefault();
+    var filename = Path.Combine(applicationData.MachinePath, "importantstuff.dat");
+    File.WriteAllText(filename, data, System.Text.UTF8Encoding);
+}
+```
+
+## 4.4. Unpackaged app using LocalPath
+
+An unpackaged app app creating a file its LocalPath data store.
+
+```c#
+void SaveData(string data)
+{
+    string publisher = "Fabrikam";
+    string product = "Contoso";
+    var applicationData = ApplicationData.GetForUnpackaged(publisher, product);
+    var filename = Path.Combine(applicationData.LocalPath, "stuff.dat");
+    File.WriteAllText(filename, data, System.Text.UTF8Encoding);
+}
+```
+
+# 5. Remarks
+
+## 5.1. Platform Support
+
+Per https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.roamingfolder#remarks:
+  > Roaming data and settings is no longer supported as of Windows 11. The recommended replacement
+      is Azure App Service. Azure App Service is widely supported, well documented, reliable, and
+      supports cross-platform/cross-ecosystem scenarios such as iOS, Android and web. Settings
+      stored here no longer roam (as of Windows 11), but the settings store is still available.
+
+We provide `RoamingFolder` and `RoamingSettings` equivalents but they're only as functional as
```

Comment:
Should we take this opportunity to just not have these on the new type? If you need those, go use the in-box type.

Reply:
Seemed reasonable to keep these 2 (we omitted all other roaming related properties/methods).

Roaming only stopped roaming after 19H2 and WinAppSDK runs down to RS5. Plus some apps use RoamingFolder and/or RoamingSettings even today (despite not roaming) 'cause compat (e.g. Tempo by @MikeHillberg).

Plus for unpackaged apps RoamingFolder === `%USERPROFILE%\AppData\Roaming` and RoamingSettings=`HKCU\SOFTWARE\...` and those are still supported.

Reply:
Personally, I'd just remove them here as "go forward" apps won't care and we don't want to give people false hope.

Reply:
Sounds good to me

RECOMMEND: Remove `RoamingFolder`, `RoamingPath` and `RoamingSettings`